psychiatric_guide_app/settings.py

```diff
@@ -143,8 +143,9 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = 'static'
+STATIC_ROOT = ''
```

Comment:
Similar to the comment below, we need constant as is in order for `python manage.py runserver` to work correctly.